plugins/steve/resource-class.js

```diff
@@ -1291,7 +1291,7 @@ export default class Resource {
           const validatorExists = Object.prototype.hasOwnProperty.call(CustomValidators, validatorName);
 
           if (!isEmpty(validatorName) && validatorExists) {
-            CustomValidators[validatorName](pathValue, this.$rootGetters, errors, validatorArgs, displayKey);
+            CustomValidators[validatorName](pathValue, this.$rootGetters, errors, validatorArgs, displayKey, data);
```

Comment:
Yes, I think we still need data, https://github.com/harvester/dashboard/blob/5e4c7cbb85b4421cf514cd9de5de026b9cf0ed4b/utils/validators/vm.js#L72,
because some of the checks use the whole data, not a property under it